homeassistant/components/sensor/wunderground.py

```diff
@@ -617,6 +618,8 @@ def _get_attributes(rest):
     'CY', 'SN', 'JI', 'YI',
 ]
 
+DEFAULT_ENTITY_NAMESPACE = 'pws'
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
```

Comment:
You are literally extending a platform schema here that includes entity namespace.

Reply:
As far as I saw, there was no `ENTITY_NAMESPACE` in the original platform schema:

https://github.com/home-assistant/home-assistant/blob/4d3743f3f79a1127208ce885ae501991ec3e704e/homeassistant/helpers/config_validation.py#L493-L496

Reply:
Hmm, there should be 🤔